packages/functional-tests/tests/subscription-tests/support.spec.ts

```diff
@@ -51,8 +51,8 @@ test.describe('severity-2 #smoke', () => {
       credentials,
     }, { project }) => {
       test.skip(
-        project.name === 'production',
-        'no real payment method available in prod'
+        project.name === 'production' || project.name === 'local-chromium',
```

Comment:
not for this PR but just thinking out loud&hellip;
I think eventually we might need a solution for excluding tests by env/browser/feature using [Playwright annotations](https://playwright.dev/docs/test-annotations). Although that might put us into weird inverted greps where all tests are for all browsers and we have to filter out certain tests that are `--grep-invert @firefox-only` for Chomium or something.

Reply:
Yes I agree, we need to figure out a more appropriate way to filter out tests.